targetplatforms/r2018-12-release.tpd

```diff
@@ -50,6 +50,7 @@ location papyrus "http://download.eclipse.org/modeling/mdt/papyrus/updates/relea
 	org.eclipse.papyrus.sdk.feature.feature.group
 	org.eclipse.papyrus.infra.gmfdiag.feature.feature.group
 	org.eclipse.papyrus.infra.services.feature.feature.group
+	org.eclipse.papyrus.views.documentation.feature.feature.group lazy
```

Comment:
Oops, copied and pasted the `lazy` version constraint from the nightly TPD.

Reply:
Good catch! Thanks!